css/_toolbars.scss

```diff
@@ -270,7 +270,6 @@
     }
 }
 
-.always-on-top-toolbox,
 .filmstrip-toolbox {
     background-color: $newToolbarBackgroundColor;
     box-sizing: border-box;
```

Comment:
Shouldn't we remove all `i` style from here?

Reply:
Also I guess the filmstrip only toolbar is now having the same issues as the AOT toolbar. Did you test?